{{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/plugin.py

```diff
@@ -45,35 +49,78 @@
 # INITIALIZATION TASKS
 ########################################
 
-# To run the script from templates/{{ cookiecutter.plugin_name }}/tasks/myservice/init, add:
-# hooks.Filters.COMMANDS_INIT.add_item((
-#     "myservice",
-#     ("{{ cookiecutter.plugin_name }}", "tasks", "myservice", "init"),
-# ))
+# To add a custom initialization task, create a bash script template under:
+#   {{ cookiecutter.module_name }}/templates/{{ cookiecutter.plugin_name }}/jobs/init/
+# and then add it to the MY_INIT_TASKS list. Each task is in the format:
+#   ("<service>", ("<path>", "<to>", "<script>", "<template>"))
+MY_INIT_TASKS: list[tuple[str, tuple[str, ...]]] = [
+    ("lms", ("{{ cookiecutter.plugin_name }}", "jobs", "init", "lms.sh")),
```

Comment:
We should probably not have these init tasks in the plugin by default. First it might be confusing for the developer. Second, running every one of these tasks will take a handful of seconds, even if the task doesn't do anything. (that's because running the lms/cms containers triggers a restart of the permission services)